deploy/charts/emqx-operator/templates/controller-manager.yaml

```diff
@@ -67,6 +69,16 @@ spec:
         - mountPath: /tmp/k8s-webhook-server/serving-certs
           name: cert
           readOnly: true
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=10
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
Maybe we can be made configurable the image via `values.yaml`, like https://github.com/emqx/emqx-operator/blob/e95d9141d560d275110054aecb6f35db948e0249/deploy/charts/emqx-operator/values.yaml#L10, this's more friendly to users

Reply:
Got it